### PR TITLE
Update CentralDashboard docs for 1.4

### LINKED ISSUE #2892: Update CentralDashboard docs for KF 1.4
We will need to update the pages of the CentralDashboard for 1.4.

We've included a new entry for KFServing https://github.com/kubeflow/kubeflow/pull/6085, so the docs will need to reflect this change.

/assign @kimwnasptd 
cc @kubeflow/wg-notebooks-leads

### PR DESCRIPTION
**Pull request by kimwnasptd**
closes #2892 

Updates the main docs page of the Dashboard to depict all the new web apps we have in Kubeflow.

/cc @RFMVasconcelos @StefanoFioravanzo @shannonbradshaw

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @RFMVasconcelos

**Review by RFMVasconcelos — COMMENTED** [review]
Thank you for the extended list of Kubeflow dashboard tabs @kimwnasptd ! 

We did not yet cut a `v1.3-branch` in order to allow v1.4 specific docs. 

It looks like user management is an addition only in v1.4 - I will cut the `v1.3-branch` at this stage if we all agree in the release meeting, and merge this to `master` afterwards.

/lgtm

File: content/en/docs/components/central-dash/overview.md [hunk 1]
```diff
@@ -16,19 +16,26 @@ following features:
...
-* **Artifact Store** for tracking of artifact metadata.
-* **Manage Contributors** for sharing user access across namespaces in the 
-  Kubeflow deployment.
+* **Home**:, Home, the central hub to access recent resources, active
+  experiments, and useful documentation.
+* **Notebook Servers**: To manage [Notebook servers](/docs/components/notebooks/).
+* **TensorBoards**: To manage TensorBoard servers.
+* **Models**: To manage deployed [KFServing models](/docs/components/kfserving/kfserving/).
+* **Volumes**: To manage the cluster's Volumes.
+* **Experiments (AutoML)**: To manage [Hyper Parameter tuning](/docs/components/katib/) experiments.
```
Comment (andreyvelich) [inline content/en/docs/components/central-dash/overview.md:31]:
```suggestion
* **Experiments (AutoML)**: To manage [Katib](/docs/components/katib/) experiments.
```

File: content/en/docs/components/central-dash/overview.md [hunk 2]
```diff
@@ -16,19 +16,26 @@ following features:
...
+* **Experiments (KFP)**: To manage [Kubeflow Pipelines](/docs/components/pipelines/) experiments.
```
Comment (andreyvelich) [inline content/en/docs/components/central-dash/overview.md:32]:
```suggestion
* **Experiments (KFP)**: To manage [Kubeflow Pipelines (KFP)](/docs/components/pipelines/) experiments.
```

**Review by andreyvelich — COMMENTED** [review]
Thank you for updating this @kimwnasptd!

**Comment by Talador12** [comment]
lgtm to me too

**Comment by jbottum** [comment]
please add an option on the Central Dashboard for Executions.    then /lgtm

**Comment by kimwnasptd** [comment]
@shannonbradshaw @zijianjoy I had to push a commit to address @jbottum's comment above. It's now ready for merge

**Comment by shannonbradshaw** [comment]
Thanks @kimwnasptd! 

@zijianjoy this LGTM

/lgtm

File: content/en/docs/components/central-dash/overview.md (hunk 1 above)
Comment (zijianjoy) [reply content/en/docs/components/central-dash/overview.md:31]: +1

Comment (shannonbradshaw) [reply content/en/docs/components/central-dash/overview.md]: @kimwnasptd it looks like we have some additional support for making this change. Would you mind pushing another commit?

File: content/en/docs/components/central-dash/overview.md [hunk 3]
```diff
@@ -16,19 +16,27 @@ following features:
...
 The Kubeflow UIs include the following:
 
-* **Home**, a central dashboard for navigation between the Kubeflow components.
-* **Pipelines** for a Kubeflow Pipelines dashboard.
-* **Notebook Servers** for Jupyter notebooks.
-* **Katib** for hyperparameter tuning.
-* **Artifact Store** for tracking of artifact metadata.
-* **Manage Contributors** for sharing user access across namespaces in the 
-  Kubeflow deployment.
+* **Home**:, Home, the central hub to access recent resources, active
```
Comment (zijianjoy) [inline content/en/docs/components/central-dash/overview.md:25]: NIT: there is a ',' symbol after **Home**:, should be removed.

File: content/en/docs/components/central-dash/overview.md [hunk 4]
```diff
@@ -16,19 +16,27 @@ following features:
...
+* **Notebook Servers**: To manage [Notebook servers](/docs/components/notebooks/).
+* **TensorBoards**: To manage TensorBoard servers.
+* **Models**: To manage deployed [KFServing models](/docs/components/kfserving/kfserving/).
+* **Volumes**: To manage the cluster's Volumes.
+* **Experiments (AutoML)**: To manage [Katib](/docs/components/katib/) experiments.
+* **Experiments (KFP)**: To manage [Kubeflow Pipelines (KFP)](/docs/components/pipelines/) experiments.
+* **Pipelines**: To manage KFP pipelines.
+* **Runs**: To manage KFP runs.
+* **Recurring Runs**: To manage KFP recurring runs.
+* **Artifacts**: To track MLMD artifacts.
```
Comment (zijianjoy) [inline content/en/docs/components/central-dash/overview.md:36]: Can we change "MLMD artifacts" to "ML Metadata (MLMD) artifacts of KFP"?

File: content/en/docs/components/central-dash/overview.md [hunk 5]
```diff
@@ -16,19 +16,27 @@ following features:
...
+* **Executions**: To track executions of Pipelines.
```
Comment (zijianjoy) [inline content/en/docs/components/central-dash/overview.md:37]: Can we say "To track MLMD executions of KFP"?

File: content/en/docs/components/central-dash/overview.md [hunk 6]
```diff
@@ -64,19 +72,19 @@ guide to
...
-If you didn't configure Kubeflow to integrate with an identity provider 
+If you didn't configure Kubeflow to integrate with an identity provider
 then you can port-forward directly to the Istio gateway.
 
 Port-forwarding typically does not work if any of the following are true:
 
-  * You've deployed Kubeflow on GCP using the 
-    [GCP deployment UI](/docs/gke/deploy/deploy-ui/) or the default settings 
+  * You've deployed Kubeflow on GCP using the
+    [GCP deployment UI](/docs/gke/deploy/deploy-ui/) or the default settings
```
Comment (zijianjoy) [inline content/en/docs/components/central-dash/overview.md:81]: Would you like to remove the GCP deployment UI approach? We have changed to use CLI deployment only

File: content/en/docs/components/central-dash/overview.md (hunk 4 above)
Comment (zijianjoy) [reply content/en/docs/components/central-dash/overview.md:36]: I just noticed there are two rows of Artifacts, we should remove the duplicated one here and keep the same order with central dashboard UI

File: content/en/docs/components/central-dash/overview.md (hunk 3 above)
Comment (kimwnasptd) [reply content/en/docs/components/central-dash/overview.md:25]: Nice catch, thanks! I added a commit

File: content/en/docs/components/central-dash/overview.md (hunk 4 above)
Comment (zijianjoy) [reply content/en/docs/components/central-dash/overview.md:36]: Oh you are right, I agree with your suggestion here.

**Pull request merged**